src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -184,13 +184,13 @@
     <value>Use generic event handler instances</value>
   </data>
   <data name="UseGenericEventHandlerInstancesForDelegateMessage" xml:space="preserve">
-    <value>Remove '{0}' and replace its usage with a generic EventHandler, for e.g. EventHandler&lt;T&gt;, where T is a valid EventArgs</value>
+    <value>Remove '{0}' and replace its usage with a generic EventHandler, for e.g. EventHandler&lt;T&gt;, where T is a valid EventArgs.</value>
```

Comment:
This is a false-positive, our naive algo is considering `. ` as new sentence separator which isn't the case here. Let me know if you want me to revert this change. Also let me know if we shall create a ticket?

Reply:
I'd change `e.g.` to `example` here

Reply:
Seems like a corner case, I'd let you make a call if you feel this is important enough to address.

Reply:
> I'd change e.g. to example here

Yeah, that sounds reasonable.